app/helpers/find/subject_helper.rb

```diff
@@ -20,6 +26,16 @@ def secondary_subject_options(subjects = secondary_subjects)
 
   private
 
+    PrimarySubjectInput = Struct.new(:code, :name)
+
+    def primary_subjects
+      Subject.where(type: "PrimarySubject")
+             .order(:subject_name)
+    end
+
+    # These subjects don’t currently match any courses, and so can be dropped.
+    IGNORED_SUBJECTS = ["Philosophy", "Modern Languages", "Ancient Hebrew", "Ancient Greek"].freeze
```

Comment:
Are theses still to be ignored?

Reply:
@avinhurry didn't you got you changed this?

Reply:
Yes, if you rebase they should be gone 🪄